documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+# Consent decision-making
+
+The Openverse maintainers have elected to adopt a decision-making model based on
+the
+[Sociocracy practice of "consent decision-making"](https://patterns.sociocracy30.org/consent-decision-making.html).
+We hope this will accomplish the following:
+
+- Bring expediency to decision-making by giving guidelines for how and what
+  types of feedback to give
+- Adopt more specific standards about what types of feedback must be addressed
+  before a proposal can be ratified for implementation
+- Give specificity in feedback expectations, both for who will participate in a
+  decision-making process and what responsibilities and tools they have for
+  participating
+- Reaffirm trust between contributors when evaluating proposals
+
+Historically, each of these have presented problems for the Openverse project as
+it has grown. Decision-making can drag on for months, proposal reviews get stuck
+in feedback cycles over issues that could be addressed later on, and there were
+few, if any, expectations around who would participate in a discussion or what
+the expectations of that participation was. Adopting the process described
+below, we hope, will address some of the underlying causes of these problems, in
+addition to the benefits described above.
+
+The process described below is heavily modified from the regular Sociocratic
+process. The differences come from two directions:
+
+- The needs of a largely asynchronously coordinated base of contributors
+- Internal feedback and iteration on the initial process, even after
+  modification for asynchrony
+
+> ## When to follow this process
+>
+> Whether this process is followed for any given proposal or discussion is left
+> entirely to the discretion of the people relevant to the discussion. We
+> believe that this process helps the project's contributors make expedient and
+> clear decisions. However, it should not become an obstacle that is followed
+> for every tiny decision. If a decision can efficiently be made without this
+> process, contributors are invited to do so.
+>
+> For the purposes of project planning, however, this process should be the
+> default and should be the strong preference, keeping in mind that it can be
+> made faster via [round shortcutting](#shortcutting-a-round) and the
+> [synchronous lightening process](#optional-synchronous-lightening-process).
+
+## Continuous improvement and exceptions to every rule
+
+**This process is not set in stone.** Nor is it a set of laws that must be
+followed at all times. This document uses prescriptive language filled with
+imperatives primarily to prevent the process itself from being filled with
+contingencies at every turn. However, as with every other aspect of our ever
```

Comment:
> This document uses prescriptive language filled with
imperatives primarily to prevent the process itself from being filled with
contingencies at every turn.

I'm trying to approach this document with accessibility in mind and simplifying language where possible. Might something like this work instead, without losing meaning? This is all very subjective so please don't feel like you _have to_ implement any of these types of suggestions.

> "This document uses prescriptive language to avoid confusion and uncertainty during the process."

Reply:
In my opinion no, because it's necessary to emphasise the fact that every imperative probably has a qualification (one of the few listed in this section). "This document uses prescriptive language to avoid confusion and uncertainty during the process" doesn't make sense to me as the avoidance of qualifying language throughout the rest of the document isn't to avoid confusion or uncertainty _during_ the process, but to simplify the language of the proposal so that it's actually readable.

Maybe:

> This document uses prescriptive language. However, keep in mind that broad qualifications exist and are described in this section. The document avoids qualifying every imperative in order to prevent the text from ballooning in size and complexity.

What do you think?

Reply:
Your proposed revision is much clearer to me, great!

Reply:
Coming in the next commit.